observation_portal/requestgroups/duration_utils.py

```diff
@@ -46,7 +46,7 @@ def get_instrument_configuration_duration(instrument_config_dict, instrument_nam
     return instrument_config_dict['exposure_count'] * duration_per_exposure
 
 
-def get_configuration_duration(configuration_dict, request_overheads):
+def get_configuration_duration(configuration_dict, request_overheads, include_front_padding=True):
```

Comment:
Usually we want front padding when calculating configuration duration, but not when we are updating the observation end time from the exposure start time, since exposure start is after the front padding section of time.